govcd/api_vcd_test.go

```diff
@@ -66,6 +67,14 @@ type TestConfig struct {
 		InternalIp      string `yaml:"internalIp,omitempty"`
 		EdgeGateway     string `yaml:"edgeGateway,omitempty"`
 		ExternalNetwork string `yaml:"externalNetwork,omitempty"`
+		Disk            struct {
```

Comment:
The purpose of these fields is to give users the opportunity to decide on the resources they want to use.
As such, the Size is the only field that matters here. If there is a test that needs to run operations on an existing disk, then we ask also for the name of a disk.
Everything else should be generated by the tests, as I said in my previous review. We should not ask users to provide names for the disks that our tests generate, but use names that are associated with the function names where the test is executed.
For example, in `Test_DiskCreate`, the disk will be called `Test_DiskCreateDisk` (see in this file the constants with a comment saying _Names for entities created by the tests_).